src/modules/facility/facility.service.ts

```diff
@@ -114,6 +122,8 @@ export class FacilityService {
     const midnightToday = this.dateStringTransformations.getDateStringFromDate(new Date());
 
     const capitalConversionFactorCode =
+      facilityToTranform.capitalConversionFactorCode ??
+      defaultValues.capitalConversionFactorCode[facilityToTranform.productTypeId] ??
```

Comment:
These 2 lines are exactly the same as the 2 lines below, I don't think we needed to add them